src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        age >= 0 && (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        )
+      }
+
+      <p className="Person__partner">
+        {
+          isMarried
+            ? `${partnerName} is my `
+              + `${sex === 'm'
+                ? 'wife'
```

Comment:
It's a bad practice you use nested ternary operators.
Try to fix it (you could separate logic with the both cases).